docs/Configuration.md

```diff
@@ -205,15 +275,28 @@ zero minutes, then the default value (600 seconds) is used.
 
 --------
 
-[Installation instructions]:          Installation.md
+[DB.database_host]:                   #db-section
+[DB.database_name]:                   #db-section
+[DB.password]:                        #db-section
+[DB.user]:                            #db-section
```

Comment:
Why not pointing to the right subsection directly ?

Reply:
I was thinking that the password and user headings were ambiguous and I wanted to keep things consistent for the others too. I'd forgotten that Github has a way of disambiguating the headings.

Thanks for pointing this out!

Reply:
Yes I totally agree that this is not the easiest to maintain.